packages/runtime/src/runtime.ts

```diff
@@ -1161,4 +1169,50 @@ export class Runtime {
 	sendTxAndWait(transactions: SignedTransaction[]): TxnReceipt[] {
 		return this.executeTx(transactions);
 	}
+
+	/**
+	 * Produces new block and adds it to a Map where the keys are block numbers
+	 */
+	produceBlock() {
+		let timestamp: bigint | undefined;
+		let seed: Uint8Array | undefined;
+		if (this.store.blocks.size === 0) { //create genesis block
```

Comment:
```suggestion
		if (!this.store.blocks.size) { //create genesis block
```

Reply:
I think its better to leave it zero. This way it is much clearer to the user that it is indeed empty